src/main/java/com/mattermost/integration/figma/notification/service/FileNotificationService.java

```diff
@@ -131,8 +135,11 @@ public void sendFileNotificationMessageToMM(FileCommentWebhookResponse fileComme
 
     private String sendMessageToCommentAuthor(FigmaWebhookResponse figmaWebhookResponse, Context context) {
         String token = getToken(figmaWebhookResponse, context);
-
         CommentDto comment = commentService.getCommentById(figmaWebhookResponse.getParentId(), figmaWebhookResponse.getFileKey(), token).get();
+        if (figmaWebhookResponse.getTriggeredBy().getId().equals(comment.getUser().getId())) {
+            return "";
```

Comment:
please consider to extract empty string into constant with meaningful name . This name could explain why we return here an empty string